### PR TITLE
Small optimization in render hyperscript.js

### PR DESCRIPTION
**Pull request by magikstm**
## Description
No functional change. I read code and noticed it.

## Motivation and Context
Minor. Feel free to pull or not.

## How Has This Been Tested?
Locally.

## Types of changes
Optimization.

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [ ] I have updated `docs/change-log.md`

### CODE REVIEW

File: render/hyperscript.js [hunk 1]
```diff
@@ -108,6 +104,7 @@ function hyperscript(selector) {
 	var normalized = Vnode.normalizeChildren(children)
 
 	if (typeof selector === "string") {
+		var cached = selectorCache[selector] || compileSelector(selector)
```
Comment (isiahmeadows) [inline render/hyperscript.js]: Mind combining this with the next line, to save a temporary variable? (Your patch is more so optimizing space than speed, BTW.)

**Comment by magikstm** [comment]
Thanks for the review. I adjusted it.

I'm having trouble to squash commits in github with the command-line tools or SourceTree.  :(

**Comment by pygy** [comment]
@magikstm The "Sloppy mode" comment was paired with the line you moved. I don't even remember why it's there (I used to, I thought I had made it more meaningful but that PR was probably not merged).

**Comment by isiahmeadows** [comment]
@pygy The sloppy mode comment was for when the bundler hadn't yet been
patched to emit strict mode bundles instead. Also, since we no longer
support IE9 in `next` (MS dropped support with Vista), we can safely assume
all browsers interpret strict mode directives correctly.

So dropping it is of no concern anymore. (I'm the one who wrote that patch
to repair sloppy mode use, BTW.)

**Comment by magikstm** [comment]
Thanks. I just removed the comment.

**Comment by isiahmeadows** [comment]
@magikstm If you're okay working from the CLI, there's `git log` to display info about commits, and `git rebase ${ref} -i`/`git rebase ${ref} --interactive` to interactively modify the history. I do that quite frequently when working with patch sets to trim the commits down to something a little more logical and workable.

**Comment by pygy** [comment]
@isiahmeadows I'm not against dropping IE 9 (and 10) support, given how marginal it has become, but I didn't know this was planned.

OTOH given the `onpopstate` bugs in IE 11 (IIRC) I'm not sure it buys us much.

**Comment by magikstm** [comment]
@isiahmeadows I'm unfortunately a bit limited with the Git CLI. I use it a bit, but usually use SourceTree. I'm not that used to squashing commits. It's a feature of Git I have to learn more.

Could you do it before a merge or after it?

**Comment by isiahmeadows** [comment]
@pygy MS dropped it with Vista, which EOL'd this year IIRC. We usually
don't bother supporting obscure (e.g. IE7 for Windows POS Ready) or
manufacturer-unsupported operating systems, and that's also why we dropped
IE 6-8 support for 1.x.

Windows 7 isn't EOL until 2020, so it's going to be a while before that
hits.

@magikstm Don't worry too much about the history - some of our own PRs have
been a bit messy history-wise. ;-)

**Pull request merged**